sys/include/suit/v3/suit.h

```diff
@@ -112,36 +125,36 @@ enum {
 };
 
 /**
- * @brief SUIT v4 component struct
+ * @brief SUIT v3 component struct
```

Comment:
Here too, how about being precise in the comments/doc and specify draft-ietf-suit-manifest-03 instead of just "v3" ?